numpy/distutils/fcompiler/pg.py

```diff
@@ -54,10 +56,57 @@ def get_flags_linker_so(self):
     def runtime_library_dir_option(self, dir):
         return '-R"%s"' % dir
 
+
+class PGroupFlangCompiler(PGroupFCompiler):
+    compiler_type = 'flang'
+    description = 'Portland Group Fortran LLVM Compiler'
+    version_pattern =  r'\s*(flang|clang) version (?P<version>[\d.-]+).*'
+
+    ar_exe = 'lib.exe'
+    possible_executables = ['flang']
+
+    executables = {
+        'version_cmd'  : ["<F77>", "--version"],
+        'compiler_f77' : ["flang"],
+        'compiler_fix' : ["flang"],
+        'compiler_f90' : ["flang"],
+        'linker_so'    : [None],
+        'archiver'     : [ar_exe, "/verbose", "/OUT:"],
+        'ranlib'       : None
+        }
+
+    compile_switch = '-nomp'
+    object_switch = '-o '     # No space after /Fo!
```

Comment:
Once thing I'm not clear about is that the object switch and the compile switch are the same for flang. I just put a fake compile switch here (pretty harmless option).

Reply:
Isn't it `-c` ?

Reply:
See https://ci.appveyor.com/project/xoviat/scipy/build/1.0.23#L3924

Reply:
`-c` is given twice here, `"C:\Miniconda36-x64\Library\bin\flang.exe -Ic:\python36-x64\lib\site-packages\numpy\core\include -c -c  scipy\fftpack\src\dfftpack\dcosqb.f -obuild\temp.win-amd64-3.6\scipy\fftpack\src\dfftpack\dcosqb.o"`